include/DeviceList/Model.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+** Project FileShare, 2022
+**
+** Author Léo Lhuile
+**
+** Started on  Sat Jan 15 21:08:44 2022 Léo Lhuile
+** Last update Wed Sep 14 22:26:27 2022 Léo Lhuile
+**
+** DeviceList/Model.hpp : Class to manage the device list data
+*/
+
+#pragma once
+
+#include <vector>
+#include <string>
+
+namespace FileShare::GUI::DeviceList {
+    class Model {
+        public:
+            Model();
+            ~Model();
+
+            void setCurrentDevice(const std::string& device) { this->currentDevice = device; }
+            std::string getCurrentDevice() { return this->currentDevice; }
+
+            void toggleCurrentDeviceConnected() { this->currentDeviceConnected = !this->currentDeviceConnected; }
```

Comment:
Instead of a toggle, I think a `setCurrentDeviceConnected` would be better ?
Toggles can get out of sync, if you set the value then it will always be correct.

Reply:
In this specific use-case, I cannot implement setCurrentDeviceConnected because the Controller is not supposed to be aware of the data in the Model. I plan to create one later when the Model is connected to the server and thus has "real" data.